src/vinyldns/batch_change.py

```diff
@@ -55,18 +55,21 @@ class BatchChangeRequest(object):
         'DeleteRecordSet': DeleteRecordSet.from_dict
     }
 
-    def __init__(self, changes, comments=None, owner_group_id=None):
+    def __init__(self, changes, comments=None, owner_group_id=None,
+                 scheduled_time=None):
         self.comments = comments
         self.changes = changes
         self.owner_group_id = owner_group_id
+        self.scheduled_time = scheduled_time
 
     @staticmethod
     def from_dict(d):
         return BatchChangeRequest(
             comments=d.get('comments'),
             changes=[BatchChangeRequest.change_type_converters[elem['changeType']](elem)
                      for elem in d.get('changes', [])],
-            owner_group_id=d.get('ownerGroupId')
+            owner_group_id=d.get('ownerGroupId'),
+            scheduled_time=d.get('scheduled_time')
```

Comment:
one thing I am not sure of is if we automatically convert to utc time.  That is in serdes.py

Reply:
mentioned here https://github.com/vinyldns/vinyldns-python/pull/56/files#r310124796